### ISSUE TITLE
Travis CI build job #321.2 has failed

### ISSUE THREAD

**Issue by rcjsuen** [rcjsuen/dockerfile-language-server-nodejs#219]
For whatever reasons, test `issue #218` got run twice [in the build](https://travis-ci.org/rcjsuen/dockerfile-language-server-nodejs/jobs/367104973). The [other](https://travis-ci.org/rcjsuen/dockerfile-language-server-nodejs/jobs/367104972) [two](https://travis-ci.org/rcjsuen/dockerfile-language-server-nodejs/jobs/367104974) builds didn't have this problem though.
```
$ npm run nyc-ci

> dockerfile-language-server-nodejs@0.0.17 nyc-ci /home/travis/build/[secure]/dockerfile-language-server-nodejs
> nyc --cache false mocha out/test



  Dockerfile LSP Tests
    ✓ initialize (883ms)
    ✓ initialized
    ✓ issue #183
    ✓ issue #209
    ✓ issue #202
    ✓ issue #214
    ✓ issue #216
    ✓ issue #207
    1) issue #218
    ✓ issue #218


  9 passing (948ms)
  1 failing

  1) Dockerfile LSP Tests issue #218:
      Uncaught AssertionError [ERR_ASSERTION]: 0 == 1
      + expected - actual

      -0
      +1
      
      at ChildProcess.listener218 (out/test/server.test.js:345:28)
      at emit (internal/child_process.js:772:12)
      at _combinedTickCallback (internal/process/next_tick.js:141:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)




  9 passing (953ms)
  1 failing

  1) Dockerfile LSP Tests issue #218:

      Uncaught AssertionError [ERR_ASSERTION]: 0 == 1
      + expected - actual

      -0
      +1
      
      at ChildProcess.listener218 (out/test/server.test.js:345:28)
      at emit (internal/child_process.js:772:12)
      at _combinedTickCallback (internal/process/next_tick.js:141:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



----------|----------|----------|----------|----------|----------------|
File      |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
----------|----------|----------|----------|----------|----------------|
All files |  Unknown |  Unknown |  Unknown |  Unknown |                |
----------|----------|----------|----------|----------|----------------|
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dockerfile-language-server-nodejs@0.0.17 nyc-ci: `nyc --cache false mocha out/test`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the dockerfile-language-server-nodejs@0.0.17 nyc-ci script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2018-04-16T11_19_56_017Z-debug.log


The command "npm run nyc-ci" exited with 1.
```

**Comment by rcjsuen** [comment]
Hopefully fixed by 3083f51108b5e5ddfd440e6fe3da415d10b9c69c.

**Comment by rcjsuen** [comment]
Looks like both [324.2 and 324.3](https://travis-ci.org/rcjsuen/dockerfile-language-server-nodejs/builds/378091770) has failed because of this error. I suppose the good thing is that it's failing because of a single run of `test #218` and not a strange "second run".
